src/main/scala/com/linkedin/feathr/offline/source/dataloader/hdfs/FileFormat.scala

```diff
@@ -71,7 +71,11 @@ object FileFormat {
         ss.read.format(ORC_DATASOURCE).load(existingHdfsPaths: _*)
       case PARQUET =>
         ss.read.parquet(existingHdfsPaths: _*)
-      case _ => throw new FeathrException(s"Unsupported data format $format. Only AVRO and ORC are supported.")
+      case _ =>
+        // if we cannot detect the file type by extension, we will detect "spark.feathr.inputFormat" and use that as the option; this is a global config (i.e. affecting all the files) so customers should use it as the last resort.
+        // If this is not set, throw an exception
+        if (ss.conf.get("spark.feathr.inputFormat", "").nonEmpty) ss.read.format(ss.conf.get("spark.feathr.inputFormat")).load(existingHdfsPaths: _*)
```

Comment:
the naming is not consistent with output.format

Reply:
It should be consistent, i.e. spark.feathr.inputFormat and spark.feathr.outputFormat